test/devise/async/model_test.rb

```diff
@@ -81,18 +101,24 @@ module Async
         after { I18n.locale = nil }
 
         it "should set the current locale to the args" do
-          admin = create_admin
-          Worker.expects(:enqueue).with(:confirmation_instructions, "Admin", admin.id.to_s, instance_of(String), { 'locale' => :de })
-          admin.send_confirmation_instructions
+          Admin.transaction do
```

Comment:
I don't get why this example and the next one you adapted look so different. The only difference should be the (non-) existence of the locale in the args.